autoprecompiles/src/symbolic_machine_generator.rs

```diff
@@ -130,22 +123,29 @@ pub fn statements_to_symbolic_machine<A: Adapter>(
         // To simplify constraint solving, we constrain `is_valid` to be 1, which effectively
         // removes the column. The optimized precompile will then have to be guarded by a new
         // `is_valid` column.
-        let minus_is_valid: AlgebraicExpression<_> = exec_receive(
+        let first_exec_mult = first_exec_interaction(
             &machine,
             bus_map.get_bus_id(&BusType::ExecutionBridge).unwrap(),
         )
         .mult
         .clone();
+        // We don't know whether the first execution bus interaction is a send or receive
+        // (this might depend on the zkVM), but we know its multiplicity should be either 1 or -1.
+        // Because we expect `is_valid` to also be bit-constrained, the solver will be able to
+        // determine the correct value.
```

Comment:
The correct value of what?

Reply:
I guess of the multiplicity

Reply:
I'll undo the change for now and open a separate PR if needed.